docs/concepts/vega-chain/network.md

```diff
@@ -1,27 +1,178 @@
 ---
-sidebar_position: 5
+sidebar_position: 3
 title: Network
 vega_network: TESTNET
 hide_title: false
 ---
 
+import NetworkParameter from '@site/src/components/NetworkParameter';
+import Topic from '/docs/topics/_topic-staking.mdx'
+
+The Vega network is operated by a number of independent validators, who each run a [validator node](validator-nodes.md), and may also run [data nodes](data-nodes.md).
+
 ## Network life
-Vega networks will, at least initially, run for a limited time only. 
+Vega networks will, at least initially, run for a limited time only, and chain restarts will pick up from the most recent valid state. A network restart will be driven by situations including Vega protocol and Tendermint software upgrades.
+
+Ensuring that the latest software versions are being used by all validators when the software is ready lessens the risk of incompatible or outdated code leading to a consensus failure, and removes the burden of supporting multiple versions. Upgrades to a running chain need to respect and be able to recalculate the deterministic state for earlier blocks, so all versions of critical code must remain in the system. 
+
+## Network-wide limits
+Some limits have been introduced to the protocol in an aim to keep the overall system performant and responsive, with low-latency. As the system relies on both a lean core and a data node that consumes and provides data, having limits allows the option to somewhat control how many computations and how much data is generated, while also allowing full use of the protocol's functionality. 
+
+These limits are adjustable, so each network can be set up for maximum flexibility, and offer tokenholders the chance to change the values of those limits through network parameter governance.
+
+These limits are not totally dissimilar to the spam controls that are also implemented, which aim to mitigate the possibility of malicious actors deliberately flooding the system with junk requests. 
+
+### Maximum pegged orders and liquidity commitment shapes
+Currently, two transaction types can be limited:
+* Total number of pegged orders on all active markets: controlled by network parameter <NetworkParameter frontMatter={frontMatter} param="limits.markets.maxPeggedOrders" />
+* Number of offsets in each liquidity commitment order shape (per side), controlled by <NetworkParameter frontMatter={frontMatter} param="market.liquidityProvision.shapes.maxSize" />
+
+If either parameter's value is decreased (through a governance proposal and vote), then the change does not affect existing orders on the market, but only new orders/liquidity commitments placed after the change is enacted. 
+
+## Spam protection
```

Comment:
with all this talk about spam protection is it worth mentioning the spam statistics API? are they mentioned elsewhere? is it mentioned anywhere what guarantees does the wallet give you with regard to spam?

Reply:
as discussed off gh but here for posterity - I've added a tip box about the spam stats API, and will cover the wallet enhancements once they're merged